server/graphql/v2/object/Order.js

```diff
@@ -142,6 +145,23 @@ export const Order = new GraphQLObjectType({
           }
         },
       },
+      membership: {
+        type: MemberOf,
+        description:
+          'This represents a MemberOf relationship (ie: Collective backed by an Individual) attached to the Order.',
+        async resolve(order) {
+          const fromAccount = await order.getFromCollective();
+          const toAccount = await order.getCollective();
+
+          return models.Member.findOne({
+            where: {
+              MemberCollectiveId: fromAccount.id,
+              CollectiveId: toAccount.id,
```

Comment:
We don't have to fetch the accounts, we can directly use the order fields (1 query instead of 3):

```suggestion
              MemberCollectiveId: order.FromCollectiveId,
              CollectiveId: order.CollectiveId,
```